### PR TITLE
Bump kubevirt to v0.45.0

### PR DESCRIPTION
**Pull request by guangbochen**
> This PR cannot be merged until harvester/harvester-installer#144 merged.

Bump kubevirt to v0.45.0

### Test Plan

- `make build-iso` and install Harvester. Then create a VM to see whether it works well.

### CODE REVIEW

File: deploy/charts/harvester/values.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ kubevirt-operator:
     operator:
       image:
         repository: registry.suse.com/harvester-beta/virt-operator
```
Comment (vasiliy-ul) [inline deploy/charts/harvester/values.yaml]:
```suggestion
        repository: registry.suse.com/suse/sles/15.3/virt-operator
```
hey, just dont forget to update the registry path: `harvester-beta` -> `suse/sles/15.3`

**Comment by weihanglo** [comment]
After changing to repository `registry.suse.com/suse/sles/15.3/xxx` and version `0.45.0-8.4.3`, the RBAC problem described in #1189 seems disappeared.

This PR cannot be merged until  harvester/harvester-installer#144 merged.

**Review by FrankYang0529 — APPROVED** [review]
LGTM

**Review by bk201 — APPROVED** [review]
LGTM

**Pull request merged**